src/koala/__init__.py

```diff
@@ -359,8 +359,43 @@ def correct_high_cosmics_and_defects(
         fig_size=12,
     ):
         """
-        Correct for Cosmic Rays and defects on the CCD
-        """
+  Task for correcting high cosmics and CCD defects using median values of nearby pixels.
+	As 2dFdr corrects for (the majority) of the cosmic rays, usually correct_high_cosmics = False.
+	ANGEL COMMENTS: Check, probably can be improved using MATT median running + plotting outside
+
+        Parameters
```

Comment:
I think it failed because Parameters has been tab-ed in and isn't aligned with Task for correcting... at the top